components/crypto/src/proto/mod.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub use crate::proto::schema::*;
+
+use crate::proto::schema::{Hash, PublicKey, Signature};
+use crate::{HASH_SIZE, PUBLIC_KEY_LENGTH, SIGNATURE_LENGTH};
+use exonum_proto::ProtobufConvert;
+use failure::Error;
+
+mod schema;
+#[cfg(test)]
+mod tests;
+
+impl ProtobufConvert for crate::Hash {
+    type ProtoStruct = Hash;
```

Comment:
Nit: `Hash`, `PublicKey`, etc are assumed by default to be core `crypto` types. Maybe use here and in other places `schema::Type` and `impl ProtobufConvert for Type`?